pyproject.toml

```diff
@@ -49,7 +49,7 @@ classifiers = [
 keywords = ["evaluation", "nlp", "llm"]
 dependencies = [
     # Base dependencies
-    "transformers>=4.38.0",
+    "transformers[sentencepiece,sklearn,sacrebleu]>=4.38.0",
```

Comment:
sacrebleu and sklearn are not linked to transformers only in lighteval - we use them as such in the metrics.
Why is it interesting to group them with transformers?

Reply:
Ah I see, I thought they were linked - will revert!